src/plugins/intel_gpu/src/runtime/kernels_cache.hpp

```diff
@@ -84,10 +84,10 @@ class kernels_cache {
     std::vector<std::string> batch_header_str;
 
     void get_program_source(const kernels_code& kernels_source_code, std::vector<batch_program>*) const;
-    void build_batch(const engine& build_engine, const batch_program& batch);
+    void build_batch(const engine& build_engine, const batch_program& batch, bool ignore_env_var = false);
```

Comment:
`ignore = false` seems like double negative. What about `check_env_var_for_model_caching = true`?

Reply:
This parameter is removed. Thank you.